default.py

```diff
@@ -39,7 +40,10 @@ def build_url(query):
     xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li, totalItems=2)
 
     # Get a list of the games from Lutris
-    args = settings.getSetting('lutris_executable') + ' --list-games --json'
+    if settings.getSetting('custom_path') in [True, 'True', 'true', 1]:
+        args = settings.getSetting('lutris_executable') + ' --list-games --json'
+    else:
+        args = args = find_executable("lutris") + ' --list-games --json'
```

Comment:
#20 `find_executable("lutris")`would fail if it doesn’t find the executable. I need to rewrite this section and add a test.